assets/javascripts/discourse/components/full-calendar-default-options.js

```diff
@@ -0,0 +1,23 @@
+import {
+  getCalendarButtonsText,
+  getCurrentBcp47Locale,
+} from "../lib/calendar-locale";
+import { buildPopover, destroyPopover } from "../lib/popover";
+
+export default function fullCalendarDefaultOptions() {
```

Comment:
would it be better to put it under `assets/javascripts/discourse/lib` folder?

it is placed in the `components` folder, but it is not a component